openstack/nova/templates/console-configmap.yaml

```diff
@@ -2,10 +2,15 @@
 {{- $cell_name := index . 1 }}
 {{- $config := index . 2 }}
 {{- with index . 0 }}
+  {{- if $config.enabled }}
 [serial_console]
+# nova-serialproxy
+serialproxy_host = 0.0.0.0
```

Comment:
This seems to be the default and we have no way to change it with this change. What's the reason we put it here? Visibility?